src/main/features/claim/views/defendant-represented.njk

```diff
@@ -32,7 +32,7 @@
         </fieldset>
 
         <div class="form-group panel panel-border-narrow js-hidden" id="company-name">
-          {{ textInput(t('Defendant representative organisation name'), 'organisationName', form) }}
+          {{ textInput(t("Representative's organisation name"), 'organisationName', form) }}
```

Comment:
In T1 we only used apostrophe’s not quote marks, we should be consistent here

i.e.
> Representative’s organisation name

Reply:
Yeah I just checked this on cross-gov and generally we should be doing it that way. I think we need to raise a ticket to do that throughout our service.

Reply:
We did it in T1 last week I think, may have forgotten to pass the message on